content/en/docs/concepts/overview/working-with-objects/common-labels.md

```diff
@@ -82,8 +82,8 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   labels:
-    app.kubernetes.io/name: myservice
-    app.kubernetes.io/instance: myservice-abcxzy
+    app.kubernetes.io/name: mydeployment
+    app.kubernetes.io/instance: mydeployment-abcxzy
```

Comment:
maybe the current label is more useful when you want to select all services and deployments using the same label selector?
Not sure how useful we apply a deployment specific label like this.

Reply:
okay, thought is was copied by mistake